pkgs/development/ocaml-modules/tsdl/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchgit, ocaml, findlib, ctypes, result, SDL2, pkgconfig, opam }:
+
+let
+  inherit (stdenv.lib) getVersion;
+
+  pname = "tsdl";
+  version = "0.9.0";
+
+in
+
+stdenv.mkDerivation {
+  name = "ocaml-${pname}-${version}";
+
+  src = fetchgit {
+    url = "http://erratique.ch/repos/tsdl.git";
+    rev = "refs/tags/v${version}";
+    sha256 = "0px6mksc9d8jlkjh7kj1g1jsak21vam2wpdna6xs8d6ihii8frnf";
+  };
+
+  buildInputs = [ ocaml findlib ctypes result pkgconfig opam ];
+  propagatedBuildInputs = [ SDL2 ];
+
```

Comment:
Given a file `a.ml` (possibly empty), trying to compile it with
`ocamlfind` produces the following error message. Hence my remark.


```
$ ocamlfind ocamlc -package tsdl a.ml
ocamlfind: Package `ctypes' not found - required by `tsdl'
```

If I don’t use `ocamlfind`, I still get similar errors:


```
No implementations provided for the following modules:
         Ctypes referenced from
/nix/store/s104m3hjlqc9nlb5zl7paqc1j6gggyh6-ocaml-tsdl-0.9.0/lib/ocaml/4.01.0/site-lib/tsdl/tsdl.cmxa(Tsdl)
```

It is weird that such errors do not occur on your side.

By the way, the `version` information in the `META` file is wrong
(`%%VERSION%%` rather than `0.9.0`).